scripts/hadoop/hadoop-install.sh

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bash
+set -e
+
+# Don't install Hadoop if requested.
```

Comment:
Can we reword this comment (after the PR goes through so we don't restart all the tests).  It's somewhat confusing to me.

Reply:
Good point, will do.

Reply:
Fixed in c6c3268.